financialinstruments/query.go

```diff
@@ -0,0 +1,55 @@
+package financialinstruments
+
+const readByUUIDStatement = `MATCH (fi:FinancialInstrument {uuid:{uuid}})
+				OPTIONAL MATCH (fi)-[:ISSUED_BY]->(org:Thing)
+				OPTIONAL MATCH (upp:UPPIdentifier)-[:IDENTIFIES]->(fi)
+				OPTIONAL MATCH (factset:FactsetIdentifier)-[:IDENTIFIES]->(fi)
+				OPTIONAL MATCH (figi:FIGIIdentifier)-[:IDENTIFIES]->(fi)
+				OPTIONAL MATCH (wsod:WSODIdentifier)-[:IDENTIFIES]->(fi)
+				return fi.uuid as uuid,
+					fi.prefLabel as prefLabel,
+					org.uuid as issuedBy,
+					{uuids:collect(distinct upp.value),
+					figiCode:figi.value,
+					factsetIdentifier:factset.value,
+					wsodIdentifier: wsod.value} as alternativeIdentifiers`
+
+const newIdentifierStatement = `MERGE (t:Thing {uuid:{uuid}})
+				CREATE (i:Identifier {value:{value}})
+				MERGE (t)<-[:IDENTIFIES]-(i)
+				set i : %s`
+
+const writeStatement = `MERGE (n:Thing{uuid: {uuid}})
+			set n={props}
+			set n :Concept
+			set n :FinancialInstrument
+			set n :Equity`
+
+const deleteEntityRelationshipsStatement = `MATCH (t:Thing {uuid:{uuid}})
```

Comment:
It depends on the final model, but should we also delete the existing organization relationship?